36823 <c-31#215> Created logger
36823 <c-31#215> appl_initialize was called
36823 <c-31#215:1> Logger was updated
36823 <c-31#215:1> Started intializing cube lib
36823 <c-31#215:1> Finished intializing cube lib with worker, starting to work
36823.1 <c-31#215:1> requestCubes => [33] Buffer size: 0
36823.1 <c-31#215:1> Sent requestCubes signal to root
36823.1 <c-31#215:1> Received 1 cubes from root
36823.1 <c-31#215:1> The main loop continues
36823.1 <c-31#215:1> Started solving a cube
37277.9 <c-31#215:1> Cube failed
37277.9 <c-31#215:1> Added failed cube
37277.9 <c-31#215:1> returnFailedCubes => [33] Buffer size: 2
37277.9 <c-31#215:1> Sent 1 failed cubes to root
37277.9 <c-31#215:1> Received receivedFailedCubes signal from root
37278 <c-31#215:1> requestCubes => [33] Buffer size: 0
37278 <c-31#215:1> Sent requestCubes signal to root
37278 <c-31#215:1> Received 1 cubes from root
37278 <c-31#215:1> The main loop continues
37278 <c-31#215:1> Started solving a cube
37815 <c-31#215:1> appl_interrupt was called
37815 <c-31#215:1> Interrupting worker
37815 <c-31#215:1> appl_interrupt was called
37815 <c-31#215:1> appl_withdraw was called
37815 <c-31#215:1> Started withdraw thread
37815 <c-31#215:1> STOP (0.00s since last cb)
37815 <c-31#215:1> Solving interrupted
37815 <c-31#215:1> The main loop continues
37815 <c-31#215:1> Leaving the main loop due to an interruption
37815 <c-31#215:1> Finished withdrawing lib
37815 <c-31#215:1> Enter destructor of CubeWorker
37815 <c-31#215:1> Time waiting for messages: 0.000
37815.1 <c-31#215:1> Enter destructor of CubeWorkerInterface
37815.1 <c-31#215:1> Enter destructor of CubeRoot
37815.1 <c-31#215:1> Finished withdraw thread
37876.3 <c-31#215:1> Enter destructor
37876.3 <c-31#215:1> Time working: 992.016
37876.3 <c-31#215:1> Time suspended: 0.000
37876.3 <c-31#215:1> Joined cleanup thread
37876.3 <c-31#215:1> Exit destructor
37876.3 <c-31#215:1> Destructing logger
